crates/rpc/Cargo.toml

```diff
@@ -10,3 +10,14 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+alloy-primitives.workspace = true
+anyhow.workspace = true
+discv5.workspace = true
```

Comment:
```suggestion
```

discv5 isn't used in this crate, could you please remove and check for other crates that are imported here but not used